fdb-extensions/src/main/java/com/apple/foundationdb/clientlog/TupleKeyCountTree.java

```diff
@@ -152,12 +152,12 @@ private synchronized void addInternal(@Nonnull List<Object> items, int itemPosit
     @Nonnull
     public TupleKeyCountTree addPrefixChild(@Nonnull Object prefix) {
         count++;
-        return children.computeIfAbsent(bytes, b -> newPrefixChild(prefix));
+        final byte[] prefixBytes = Tuple.from(prefix).pack();
+        return children.computeIfAbsent(bytes, b -> newPrefixChild(prefixBytes, prefix));
```

Comment:
```suggestion
        return children.computeIfAbsent(prefixBytes, b -> newPrefixChild(prefixBytes, prefix));
```
I think? I'm not actually sure, but I thought that's what you were alluding to in the issue text.

Reply:
Yes. I also renamed the formal parameters to not conflict with the field to cut down on such confusion.